build-shims.js

```diff
@@ -77,7 +77,8 @@ mockery.registerMock('@zimbra-client/enhancers', {
 
 mockery.registerMock('@zimbra-client/constants', {
 	ATTENDEE_ROLE: 1,
-	PARTICIPATION_STATUS: 1
+	PARTICIPATION_STATUS: 1,
+	supportedMimes: 1
```

Comment:
Why is this constant not all CAPS like the others?

Reply:
because this is the way it was already written in zm-x-web. The cli is just allowing it to pass from the webclient to the zimlet